tests/ResolveImports/Xamarin.Android.BindingResolveImportLib1/Xamarin.Android.BindingResolveImportLib1.targets

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Any way we can merge these `Xamarin.Android.BindingResolveImportLib*.targets` files into one? For the most part the `BuildJar` and `CleanJar` targets are identical.

Reply:
Done. There is now a new `build-tools/scripts/Jar.targets` file which contains a `BuildTestJarFile` target. This takes the `@(TestJarEntry)` item group -- which is assumed to contain Java source code -- compiles it with `javac`, and then `jar`s it into `%(TestJarEntry.OutputFile)`. Through the joy of Target batching, this allows a project to fairly easily generate any number of output `.jar` files without too much difficulty.